tests/jinja2/Jinja2BearTest.py

```diff
@@ -39,6 +58,16 @@ def test_variable_spacing(self):
                              '-foo {{var }} bar\n'
                              '+foo {{ var }} bar')
 
+    def test_statement_spacing(self):
+        content = (r'foo {%-statement1 %} bar',)
+        with execute_bear(self.uut, 'F', content) as result:
+            self.assertEqual(result[0].diffs['F'].unified_diff,
+                             '--- \n'
+                             '+++ \n'
+                             '@@ -1 +1 @@\n'
+                             '-foo {%-statement1 %} bar\n'
+                             '+foo {%- statement1 %} bar')
```

Comment:
could you also test with `{%-statement1%}` and `{%- statement1%}` and `{%-     statement1   %}` ? :)

Reply:
In fact you can move your `verify_local_bear` tests here ;)

Reply:
though... actually nvm, I think your tests using `verify_local_bear` are actually quite good.

Reply:
>could you also test with {%-statement1%} and {%- statement1%} and {%- statement1 %}


Done !
> In fact you can move your verify_local_bear tests here

Do i need to move the tests in `verify_local_bear` also?

Reply:
no really keep them as they are 😅 (I should have removed my comments 😅 )

Reply:
Okay Got it :)